android-project/app/src/main/java/oceantreasur/es/android_project/GameActivity.java

```diff
@@ -147,16 +126,14 @@ public void onClick(View v) {
 
```

Comment:
Rename sendRequest(), used in the onClickListener to something more meaningful (like checkAnswer).